locales/en-US/app.ftl

```diff
@@ -32,18 +32,17 @@ landingIntroOne = Test new features.
 landingIntroTwo = Give your feedback.
 landingIntroThree = Help build Firefox.
 landingLegalNotice = By proceeding, you agree to the <a>Terms of Use</a> and <a>Privacy Notice</a> of Test Pilot.
-landingExperimentsTitle = Try out the latest experimental features
 
 // Related to the installation of the Test Pilot add-on.
 [[landingInstall]]
 landingInstallButton = Install the Test Pilot Add-on
 landingInstallingButton = Installing...
-landingInstalledButton = Choose your features
 
 // Related to a one click to install test pilot and an experiment.
 [[oneClickInstall]]
 oneClickInstallMinorCta = Install Test Pilot &amp;
 oneClickInstallMajorCta = Enable {$title}
+enabledExperimentCta =  Enable {$title}
```

Comment:
Does this need to be a separately translated string? Or could we re-use `oneClickInstallMajorCta `